gobblin-admin/build.gradle

```diff
@@ -34,6 +34,7 @@ dependencies {
     compile externalDependency.jodaTime
 
     testCompile externalDependency.testng
+    testCompile externalDependency.slf4jToLog4j
```

Comment:
why this is only added for test-compile (isn't that the case we use slf4j all over the places?

Reply:
OK I need a bit more clarification here: 
I think the purpose of adding this line is to add the binding in these test classes to make the log4j being picked as the implementation of slf4j ?